src/main/java/Storage.java

```diff
@@ -0,0 +1,132 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
```

Comment:
Perhaps you can add a space between imports from different packages to makes it easier to browse the list and determine the dependencies when there are many imports